lib/core/src/lib/ui-header/ui-header.component.md

```diff
@@ -0,0 +1,132 @@
+# UI Header Component
```

Comment:
can it be just Header component? the UI stands for "user interface" and every component in the library can be related to UI

Reply:
renamed, the reason why I added 'ui' prefix is that in task requirements we have following line: `Component: adf-ui-header `